website/src/react-native/css/react-native.css

```diff
@@ -2030,6 +2030,14 @@ article li {
 .btn:hover {
   text-decoration: none !important; }
 
+@media screen and (max-width: 373px) {
+  .btn {
```

Comment:
In terms of specificity, this *might* be fine, I just have less confidence in this css change than I would if it was limited to just the buttons we are trying to fix rather than *all* buttons on small screens.

My question about the bottom margin is that if we are just trying to fix these for small screens, do we really need to be modifying things for large screens too?